src/_data/whats-new.yml

```diff
@@ -4,8 +4,26 @@ description:
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Thu May 28 15:55:01 2020
+updated: Tue Jun 9 10:53:40 2020
 entries:
+- description: Removed the Extension Manager section of the Software Update Guide
+    and redirected to [CLI installation](https://devdocs.magento.com/extensions/install/).
+  versions: 2.3.x
+  type: Major Update
+  date: June 3, 2020
+  link: https://github.com/magento/devdocs/pull/7343
+- description: Added a brief note about a new known issue with the [Extension Manager
+    showing no extensions in Magento Commerce 2.3.x](https://support.magento.com/hc/en-us/articles/360043980352).
```

Comment:
Links here should point to the updated topic on devdocs.magento.com, not external resources.

Reply:
We spoke about this internally and I gave Dani the go-ahead to link to the KB. This updated affected many files on devdocs, so I don't know how practical it is to link to all of them.

Reply:
If it's multiple files, then I am good with this link.

Reply:
yes, it was a link to the KB that appears in plenty of topics